torch/_tensor_docs.py

```diff
@@ -3291,8 +3292,15 @@ def callable(a, b) -> number
         or of the same dimensionality as ``src``. When empty, the operation
         returns ``self`` unchanged.
     src (Tensor or float): the source element(s) to scatter.
+
+Keyword arguments:
     reduce (str, optional): reduction operation to apply, can be either
         ``'add'`` or ``'multiply'``.
+    unique_indices (bool, optional): whether the indices to be updated in
```

Comment:
can you add that it's users responsibility to correctly set `unique_indices` and it's not checked if indices are really unique?

Reply:
done